src/Hal/JsonSchema/SchemaFactory.php

```diff
@@ -76,7 +76,7 @@ public function buildSchema(string $className, string $format = 'jsonhal', strin
             $schema['properties'] = [
                 '_embedded' => [
                     'type' => 'array',
-                    'items' => $items,
+                    'item' => $items,
```

Comment:
Actually this schema is wrong see: https://json-schema.org/understanding-json-schema/reference/array.html

it should be something like : 
```json
"_embedded": {
  "type": "object",
  "properties": {
      "item": {"type": "array", "items": $items }
  }
}
```

where items are the json schema of the array items. 

About `item`, indeed it's quite confusing, it should be a name like `contacts` or `dummies`. We'll try to fix this in the coming year but this will definitely break and is quite hard to do right now.

Reply:
can you fix it regarding or isn't there anything to fix?